### ISSUE TITLE
[Bug] I can't download on Xiaomi MI MIX 3

### ISSUE THREAD

**Issue by Drakon5999** [Ayagikei/LifeUp#32]
**Describe the bug**
Play market writes "This app is not available for your device."

**Expected behavior**
I expect that I can download the app.

**Environments and Version (please complete the following information):**
 - Device: Xiaomi Mi MIX 3
 - OS: 9 PKQ1.180729.001
 - Version 1.89.0-rc04

**Comment by Ayagikei** [comment]
Have you installed the v1.89.0-rc04 version but cannot download the v1.89.0-rc05 version?

Also, please confirm your Google Play account region.
Currently, **Google Play restricts the purchase of paid apps and even update downloads for Russia and Belarus from May 5th.** There is nothing we can do about it for now. :(
> https://support.google.com/googleplay/android-developer/answer/11950272

If you need a new version of the application, you can also choose to email us to get the installation package of the Pro version which will also check the Play licenses.

**Comment by Drakon5999** [comment]
No, I found this app in the search, but I couldn't install it. Apparently, this is really due to the restrictions of payment from Russia and Belarus.

There are no other ways to purchase the app, right?

Well, thanks for the answer!

**Comment by Ayagikei** [comment]
> No, I found this app in the search, but I couldn't install it. Apparently, this is really due to the restrictions of payment from Russia and Belarus.
> 
> There are no other ways to purchase the app, right?
> 
> Well, thanks for the answer!

**Currently no.**

Even if you get the app with a redemption code, Google Play will also restrict downloads and updates.

We are also following up on this policy and will check for new solutions.

**Some temporary solutions:**
1. Try to switch your Google Play account to another region;
2. If you have already purchased the app, you can contact us to get the general installation package
3. In the future, we will try to release a version on the official website
4. We recently launched the app on the Amazon App Store, but don't know if there will be other restrictions

**Comment by Ayagikei** [comment]
We have released a trial version on Github Release and our website.

This version does not depend on Google Play. If you have purchased LifeUp on Google Play. You can also email us to get the license redemption code for this version.

> Since this version does not depend on Google Play, the license needs to be bound to the login account.

**Comment by Ayagikei** [comment]
Based on recent feedback, it seems Google Play has recently made some changes or strengthened its restrictions, which might be impacting purchase verification, particularly for users in certain **Google Play restricted regions**.

This issue specifically affects users of the direct license version built from our GitHub releases who purchased the app through Google Play. It might prevent the app from correctly verifying your Google Play purchase entitlement.

If you are encountering problems accessing your purchased license, especially if you are in a region affected by potential Google Play restrictions, please contact us directly for assistance. To help us locate your purchase and restore access, please include either your Google Play order number (which typically starts with "GPA.") or the Google account email address you used for the purchase on Google Play.
